package com.dbserver.desafiovotacao.domain.exceptions;

public class AssociadoNaoEncontradoException extends RuntimeException {
    public AssociadoNaoEncontradoException(Long id) {
        super("Associado n√£o encontrado com o Id: " + id);
    }
}
